spec/controllers/post_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe PostController, type: :controller do
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #edit" do
+    it "returns http success" do
+      get :edit
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #update" do
+    it "returns http success" do
+      get :update
+      expect(response).to have_http_status(:success)
+    end
+  end
```

Comment:
Same with this, I'd expect it be a `PUT` route.  Do the tests pass?

Reply:
No they do not 😅 I would normally fix it in this PR, but the next task is to write tests for post creation, so I think it makes more sense just to do it in a separate PR so I can have one PR per task.